core/src/main/java/io/parsingdata/metal/Shorthand.java

```diff
@@ -203,4 +204,8 @@ private Shorthand() {}
         return outBytes;
     }
 
+    private static Token[] concatVarArgTokens(final Token token1, final Token token2, final Token... tokens) {
+        return Stream.concat(Stream.of(token1), Stream.concat(Stream.of(token2), Stream.of(tokens))).toArray(Token[]::new);
```

Comment:
:)

Yes, your assumption is right. The constructors of Seq and Cho check for `null` for all tokens.